clarifaipyspark/dataset.py

```diff
@@ -0,0 +1,371 @@
+import json
+import uuid
+from typing import List
+import os
+import time
+import requests
+from clarifai.client.app import App
+from clarifai.client.dataset import Dataset
+from clarifai.client.input import Inputs
+from clarifai.client.user import User
+from clarifai.errors import UserError
+from clarifai_grpc.grpc.api.resources_pb2 import Text
+from google.protobuf.json_format import MessageToJson
+from google.protobuf.struct_pb2 import Struct
+from pyspark.sql import DataFrame as SparkDataFrame
+from pyspark.sql import SparkSession
+
+
+class Dataset(Dataset):
+  """Dataset class provides information about dataset of the app and it inherits from the clarifai SDK Dataset class."""
+
+  def __init__(self, user_id: str = "", app_id: str = "", dataset_id: str = ""):
+    """Initializes the Dataset object.
+
+    Args:
+        user_id (str): The clarifai user ID of the user.
+        app_id (str): Clarifai App ID.
+        dataset_id (str): Dataset ID of the dataset inside the clarifai App.
+
+    Example: TODO
+    """
+    self.user = User(user_id=user_id)
+    self.app = App(app_id=app_id)
+    #Inputs object - for listannotations
+    #input_obj = User(user_id="user_id").app(app_id="app_id").inputs()
+    self.user_id = user_id
+    self.app_id = app_id
+    self.dataset_id = dataset_id
+    super().__init__(user_id=user_id, app_id=app_id, dataset_id=dataset_id)
+
+  def upload_dataset_from_csv(self,
```

Comment:
```suggestion
  def upload_from_csv(self,
```
this goes for all functions. since these are from `Dataset` class, I don't think `dataset` should be specified.